@kiva/kv-components/src/vue/KvWwwHeader/KvHeaderLinkBar.vue

```diff
@@ -83,17 +85,26 @@
 				:count="basketCount"
 			/>
 		</a>
-		<!-- avatar (sm, auth) -->
-		<KvUserAvatar
-			v-if="loggedIn"
-			ref="avatar"
-			class="tw-cursor-pointer tw-order-last tw-inline-flex"
-			:lender-name="lenderName"
-			:lender-image-url="lenderImageUrl"
-			is-small
-			@mouseover="onHover(avatar, KvHeaderMyKivaMenu)"
+		<div
+			class="tw-cursor-pointer tw-flex tw-items-center tw-gap-1"
+			@mouseover="handleAvatarMenuPosition"
 			@mouseout="onHover()"
-		/>
+		>
+			<span
+				v-if="loggedIn"
+				class="tw-bg-eco-green-1 tw-py-0.5 tw-px-1 tw-text-eco-green-4"
+			>
+				{{ numeral(balance).format('$0') }}
+			</span>
+			<!-- avatar (sm, auth) -->
+			<KvUserAvatar
```

Comment:
I think the profile menu needs a couple small tweaks:

For desktop, left aligned text.

For mobile, wider so there isn't word wrap. Full width is probably fine for mobile.